assets/src/stories-editor/blocks/amp-story-text/edit.css

```diff
@@ -47,6 +51,19 @@ div[data-type="amp/amp-story-page"] .block-editor-inner-blocks .block-editor-blo
 	display: inline-block;
 }
 
-.editor-styles-wrapper .wp-block-amp-story-text p.is-amp-fit-text {
+.wp-block[data-type="amp/amp-story-text"] .wp-block-amp-story-text .is-amp-fit-text {
```

Comment:
Should the same be added for the title and the author blocks as well?

Reply:
When this is proven to be working for this block, yes.

Reply:
Okay, makes sense. Right now it's still not possible to change the height of the title and author block unless `amp-fit-text` is used, I guess that's related? (The first comment within https://github.com/ampproject/amp-wp/pull/2296#issuecomment-492996501, it seems to be working OK now for the Text block but not for other text-like blocks).

Reply:
Ah, the gif in that comment was for the text block. I didn't realize that is was broken for the meta blocks.

Reply:
Done now.